testsuite/tests/typing-gadts/pr13579.ml

```diff
@@ -92,6 +92,7 @@ Error: This pattern matches values of type "($'a M.p * a, $'a M.p * a) Type.eq"
        but a pattern was expected which matches values of type
          "($'a M.p * a, 'b M.p * int) Type.eq"
        The type constructor "$'a" would escape its scope
+       Hint: "$'a" are type variables introduced by the equation "'_b" = "$'a"
```

Comment:
This is actually confusing, I believe `'_b` is a type parameter, so we would probably want to print the equation of the containing type